ksp_plugin_test/plugin_test.cpp

```diff
@@ -468,29 +484,35 @@ TEST_F(PluginTest, HierarchicalInitialization) {
       /*celestial_index=*/1,
       /*parent_index=*/0,
       elements,
-      make_not_null_unique<MassiveBody>(2 * SIUnit<GravitationalParameter>()));
+      make_not_null_unique<RotatingBody<Barycentric>>(
+          2 * SIUnit<GravitationalParameter>(),
+          body_rotation_));
   elements.semimajor_axis = 1 * Metre;
   plugin_->InsertCelestialJacobiKeplerian(
       /*celestial_index=*/2,
       /*parent_index=*/0,
       elements,
-      make_not_null_unique<MassiveBody>(1 * SIUnit<GravitationalParameter>()));
+      make_not_null_unique<RotatingBody<Barycentric>>(
+          1 * SIUnit<GravitationalParameter>(),
+          body_rotation_));
   elements.mean_anomaly = π * Radian;
   plugin_->InsertCelestialJacobiKeplerian(
       /*celestial_index=*/3,
       /*parent_index=*/1,
       elements,
-      make_not_null_unique<MassiveBody>(1 * SIUnit<GravitationalParameter>()));
+      make_not_null_unique<RotatingBody<Barycentric>>(
+          1 * SIUnit<GravitationalParameter>(),
+          body_rotation_));
   EXPECT_CALL(*mock_ephemeris_, WriteToMessage(_))
       .WillOnce(SetArgPointee<0>(valid_ephemeris_message_));
   plugin_->EndInitialization();
   EXPECT_CALL(*mock_ephemeris_, Prolong(_)).Times(AnyNumber());
   EXPECT_THAT(plugin_->CelestialFromParent(1).displacement().Norm(),
-              AlmostEquals(3.0 * Metre, 1));
+              AlmostEquals(3.0 * Metre, 3));
   EXPECT_THAT(plugin_->CelestialFromParent(2).displacement().Norm(),
-              Eq(1 * Metre));
+              AlmostEquals(1 * Metre, 2, 3));
```

Comment:
This seems wrong.  We should only use the 3-argument `AlmostEquals` in a loop, I think.